Build and fixed compiled errors with respect to neo4j 1.6-SNAPSHOT version + Other Change

nrkkalyan opened the pull request:
Hi,

I am new to the world of plugin development. However I tried to add few functionalities.
Please review and give feedback. 

I will be glad to here from you.

I have modified 2 features.

1> Added support for 1.6-Snapshot
2> Added icon to choose database location in the main screen beside preferences icon.

Kalyan

nrkkalyan committed:
Removed old jar files

nrkkalyan committed:
Updated the project to 1.6-Snapshot

nrkkalyan committed:
Added database choose option in the main screen beside preferences icon.
Some minor refactor due to Neo4J 1.6-Snapshot

nawroth commented:
Nice to see a contribution in here :-)
Two comments:

I see removal of jar files, but no addition of new ones?

The diffs would be much better to read if you don't re-format existing code. You could use the Neo4j formatting template, found here: http://neo4j.github.com/

nrkkalyan commented:
I have added only one new jar files in the 3rd commit as I felt other jar files are not necessary.
Please let me know if I am wrong.  

When I removed the old jars I didn't find any problems when running the application. 
I will surely use the formatting template.

Please let me know if I should revert any of the commits.

nawroth commented:
If you package it in this state, it won't work at all I think (running it from inside Eclipse IDE is a different story). However, for sure some features will be gone. The easiest route is to just keep the same set of jar files around but from the new version.

Could you commit after using the formatter and add that commit to the pull request? The code has previously been formatted using the Eclipse formatter.

nrkkalyan committed:
Revert "Removed old jar files"

This reverts commit 34e1c9931dc9caea295ea7afa40e487b37bc2c73.

nrkkalyan committed:
Revert "Updated the project to 1.6-Snapshot"

This reverts commit 0408069bc56ab1801e668e6271e51da38615b2d9.

nrkkalyan committed:
Revert "Added database choose option in the main screen beside preferences icon."

This reverts commit cde31c4ac97f450ba8204022d651c4a94e226af0.

nrkkalyan committed:
Upgraded to neo4j 1.6-SNAPSHOT, Removed Database Location from preferences  and created as its own icon beside start button, display database location in the status upon DB selection.

nrkkalyan commented:
Reverted all my previous changes and updated accordingly using the formatting template, 
Please review the changes and merge if you find ok. 

Will appreciate your feedback.

nrkkalyan committed:
Fixed build path issues

nrkkalyan commented:
Fixed classpath  in 'org.neo4j.neoclipse.doc' according to 1.6-Snapshot

peterneubauer commented:
Guys,
I am trying to run the eclipse.product with Eclipse Indigo RCP, GEF-ZEST installed, but get the stack trace below. any idea what is wrong?

[12/17/11 5:06:00 PM] Peter Neubauer: !MESSAGE Missing required bundle org.eclipse.ui_[3.6.0,4.0.0).
!SUBENTRY 2 org.eclipse.ui.cheatsheets 2 0 2011-12-17 17:04:50.058
!MESSAGE Missing optionally required bundle org.eclipse.help.base_[3.5.0,4.0.0).
!SUBENTRY 2 org.eclipse.ui.cheatsheets 2 0 2011-12-17 17:04:50.058
!MESSAGE Missing optionally required bundle org.eclipse.help.ui_[3.5.0,4.0.0).
!SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.058
!MESSAGE Bundle org.eclipse.compare_3.5.201.R37x_v20110817-0800 [51] was not resolved.
!SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.058
!MESSAGE Missing required bundle org.eclipse.ui_[3.5.0,4.0.0).
!SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.059
!MESSAGE Missing required bundle org.eclipse.ui.ide_[3.3.0,4.0.0).
!SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.059
!MESSAGE Missing required bundle org.eclipse.ui.views_[3.2.0,4.0.0).
!SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.059
!MESSAGE Missing required bundle org.eclipse.ui.workbench.texteditor_[3.5.0,4.0.0).
!SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.059
!MESSAGE Missing required bundle org.eclipse.ui.editors_[3.5.0,4.0.0).
!SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.059
!MESSAGE Missing required bundle org.eclipse.compare.core_[3.5.0,4.0.0).
!SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.059
!MESSAGE Bundle org.eclipse.zest.core_1.3.0.v20110221-2050 [52] was not resolved.
!SUBENTRY 2 org.eclipse.zest.core 2 0 2011-12-17 17:04:50.060
!MESSAGE Missing required bundle org.eclipse.ui_0.0.0.
!SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.060
!MESSAGE Bundle org.junit4_4.8.1.v20100525 [53] was not resolved.
!SUBENTRY 2 org.junit4 2 0 2011-12-17 17:04:50.060
!MESSAGE Missing required bundle org.hamcrest.core_1.1.0.
!SUBENTRY 2 org.junit4 2 0 2011-12-17 17:04:50.060
!MESSAGE Missing required bundle org.junit_4.8.1.
!SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.060
!MESSAGE Bundle org.eclipse.ui_3.7.0.I20110602-0100 [58] was not resolved.
!SUBENTRY 2 org.eclipse.ui 2 0 2011-12-17 17:04:50.060
!MESSAGE Missing required bundle org.eclipse.ui.workbench_[3.5.0,4.0.0).
!SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.061
!MESSAGE Bundle org.apache.lucene_2.9.1.v201101211721 [60] was not resolved.
!SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.061
!MESSAGE Missing required bundle org.apache.lucene.core_[2.9.1,3.0.0).
!SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.061
!MESSAGE Missing optionally required bundle org.apache.lucene.analysis_[2.9.1,3.0.0).
!SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.061
!MESSAGE Missing optionally required bundle org.apache.lucene.highlighter_[2.9.1,3.0.0).
!SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.061
!MESSAGE Missing optionally required bundle org.apache.lucene.memory_[2.9.1,3.0.0).
!SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.061
!MESSAGE Missing optionally required bundle org.apache.lucene.queries_[2.9.1,3.0.0).
!SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.062
!MESSAGE Missing optionally required bundle org.apache.lucene.snowball_[2.9.1,3.0.0).
!SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.062
!MESSAGE Missing optionally required bundle org.apache.lucene.spellchecker_[2.9.1,3.0.0).
!SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.062
!MESSAGE Missing optionally required bundle org.apache.lucene.misc_[2.9.1,3.0.0).
!SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.062
!MESSAGE Bundle org.eclipse.ui.workbench.texteditor_3.7.0.v20110505-0800 [61] was not resolved.
!SUBENTRY 2 org.eclipse.ui.workbench.texteditor 2 0 2011-12-17 17:04:50.062
!MESSAGE Missing required bundle org.eclipse.compare.core_[3.5.0,4.0.0).
!SUBENTRY 2 org.eclipse.ui.workbench.texteditor 2 0 2011-12-17 17:04:50.062
!MESSAGE Missing required bundle org.eclipse.ui_[3.5.0,4.0.0).
!SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.063
!MESSAGE Bundle org.eclipse.ui.views.properties.tabbed_3.5.200.I20110201-0800 [64] was not resolved.
!SUBENTRY 2 org.eclipse.ui.views.properties.tabbed 2 0 2011-12-17 17:04:50.063
!MESSAGE Missing required bundle org.eclipse.ui.views_[3.2.0,4.0.0).
!SUBENTRY 2 org.eclipse.ui.views.properties.tabbed 2 0 2011-12-17 17:04:50.063
!MESSAGE Missing required bundle org.eclipse.ui_[3.3.0,4.0.0).
!SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.063
!MESSAGE Bundle org.eclipse.platform_3.7.1.v201109091335 [67] was not resolved.
!SUBENTRY 2 org.eclipse.platform 2 0 2011-12-17 17:04:50.063
!MESSAGE Missing required bundle org.eclipse.ui.intro_[3.2.0,4.0.0).
!SUBENTRY 2 org.eclipse.platform 2 0 2011-12-17 17:04:50.063
!MESSAGE Missing optionally required bundle org.eclipse.ui.cheatsheets_[3.2.0,4.0.0).
!SUBENTRY 2 org.eclipse.platform 2 0 2011-12-17 17:04:50.064
!MESSAGE Missing optionally required bundle org.eclipse.ui_[3.2.0,4.0.0).
!SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.064
!MESSAGE Bundle org.apache.lucene.analysis_2.9.1.v201101211721 [71] was not resolved.
!SUBENTRY 2 org.apache.lucene.analysis 2 0 2011-12-17 17:04:50.064
!MESSAGE Missing required bundle org.apache.lucene.core_[2.9.1,3.0.0).
!SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.064
!MESSAGE Bundle org.eclipse.ltk.ui.refactoring_3.6.0.v20110505-0800 [78] was not resolved.
!SUBENTRY 2 org.eclipse.ltk.ui.refactoring 2 0 2011-12-17 17:04:50.064
!MESSAGE Missing required bundle org.eclipse.ui_[3.5.0,4.0.0).
!SUBENTRY 2 org.eclipse.ltk.ui.refactoring 2 0 2011-12-17 17:04:50.064
!MESSAGE Missing required bundle org.eclipse.ui.navigator_[3.3.200,4.0.0).
!SUBENTRY 2 org.eclipse.ltk.ui.refactoring 2 0 2011-12-17 17:04:50.065
!MESSAGE Missing required bundle org.eclipse.compare_[3.5.0,4.0.0).
!SUBENTRY 2 org.eclipse.ltk.ui.refactoring 2 0 2011-12-17 17:04:50.065
!MESSAGE Missing required bundle org.eclipse.team.ui_[3.4.100,4.0.0).

!ENTRY org.eclipse.osgi 4 0 2011-12-17 17:04:50.069
!MESSAGE Application error
!STACK 1
java.lang.RuntimeException: No application id has been found.
    at org.eclipse.equinox.internal.app.EclipseAppContainer.startDefaultApp(EclipseAppContainer.java:242)
    at org.eclipse.equinox.internal.app.MainApplicationLauncher.run(MainApplicationLauncher.java:29)
    at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:110)
    at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:79)
    at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:344)
    at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:179)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
    at java.lang.reflect.Method.invoke(Method.java:597)
    at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:622)
    at org.eclipse.equinox.launcher.Main.basicRun(Main.java:577)
    at org.eclipse.equinox.launcher.Main.run(Main.java:1410)
    at org.eclipse.equinox.launcher.Main.main(Main.java:1386)
An error has occurred. See the log file
/Users/peterneubauer/Documents/workspaces/neoclipse/.metadata/.plugins/org.eclipse.pde.core/neoclipse.product/1324137888703.log.

nrkkalyan commented:
Are you trying after my commits?

On Sun, Dec 18, 2011 at 2:24 PM, Peter Neubauer <
reply@reply.github.com

> wrote:
> 
> Guys,
> I am trying to run the eclipse.product with Eclipse Indigo RCP, GEF-ZEST
> installed, but get the stack trace below. any idea what is wrong?
> 
> [12/17/11 5:06:00 PM] Peter Neubauer: !MESSAGE Missing required bundle
> org.eclipse.ui_[3.6.0,4.0.0).
> !SUBENTRY 2 org.eclipse.ui.cheatsheets 2 0 2011-12-17 17:04:50.058
> !MESSAGE Missing optionally required bundle
> org.eclipse.help.base_[3.5.0,4.0.0).
> !SUBENTRY 2 org.eclipse.ui.cheatsheets 2 0 2011-12-17 17:04:50.058
> !MESSAGE Missing optionally required bundle
> org.eclipse.help.ui_[3.5.0,4.0.0).
> !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.058
> !MESSAGE Bundle org.eclipse.compare_3.5.201.R37x_v20110817-0800 [51] was
> not resolved.
> !SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.058
> !MESSAGE Missing required bundle org.eclipse.ui_[3.5.0,4.0.0).
> !SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.059
> !MESSAGE Missing required bundle org.eclipse.ui.ide_[3.3.0,4.0.0).
> !SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.059
> !MESSAGE Missing required bundle org.eclipse.ui.views_[3.2.0,4.0.0).
> !SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.059
> !MESSAGE Missing required bundle
> org.eclipse.ui.workbench.texteditor_[3.5.0,4.0.0).
> !SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.059
> !MESSAGE Missing required bundle org.eclipse.ui.editors_[3.5.0,4.0.0).
> !SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.059
> !MESSAGE Missing required bundle org.eclipse.compare.core_[3.5.0,4.0.0).
> !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.059
> !MESSAGE Bundle org.eclipse.zest.core_1.3.0.v20110221-2050 [52] was not
> resolved.
> !SUBENTRY 2 org.eclipse.zest.core 2 0 2011-12-17 17:04:50.060
> !MESSAGE Missing required bundle org.eclipse.ui_0.0.0.
> !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.060
> !MESSAGE Bundle org.junit4_4.8.1.v20100525 [53] was not resolved.
> !SUBENTRY 2 org.junit4 2 0 2011-12-17 17:04:50.060
> !MESSAGE Missing required bundle org.hamcrest.core_1.1.0.
> !SUBENTRY 2 org.junit4 2 0 2011-12-17 17:04:50.060
> !MESSAGE Missing required bundle org.junit_4.8.1.
> !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.060
> !MESSAGE Bundle org.eclipse.ui_3.7.0.I20110602-0100 [58] was not resolved.
> !SUBENTRY 2 org.eclipse.ui 2 0 2011-12-17 17:04:50.060
> !MESSAGE Missing required bundle org.eclipse.ui.workbench_[3.5.0,4.0.0).
> !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.061
> !MESSAGE Bundle org.apache.lucene_2.9.1.v201101211721 [60] was not
> resolved.
> !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.061
> !MESSAGE Missing required bundle org.apache.lucene.core_[2.9.1,3.0.0).
> !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.061
> !MESSAGE Missing optionally required bundle
> org.apache.lucene.analysis_[2.9.1,3.0.0).
> !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.061
> !MESSAGE Missing optionally required bundle
> org.apache.lucene.highlighter_[2.9.1,3.0.0).
> !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.061
> !MESSAGE Missing optionally required bundle
> org.apache.lucene.memory_[2.9.1,3.0.0).
> !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.061
> !MESSAGE Missing optionally required bundle
> org.apache.lucene.queries_[2.9.1,3.0.0).
> !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.062
> !MESSAGE Missing optionally required bundle
> org.apache.lucene.snowball_[2.9.1,3.0.0).
> !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.062
> !MESSAGE Missing optionally required bundle
> org.apache.lucene.spellchecker_[2.9.1,3.0.0).
> !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.062
> !MESSAGE Missing optionally required bundle
> org.apache.lucene.misc_[2.9.1,3.0.0).
> !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.062
> !MESSAGE Bundle org.eclipse.ui.workbench.texteditor_3.7.0.v20110505-0800
> [61] was not resolved.
> !SUBENTRY 2 org.eclipse.ui.workbench.texteditor 2 0 2011-12-17 17:04:50.062
> !MESSAGE Missing required bundle org.eclipse.compare.core_[3.5.0,4.0.0).
> !SUBENTRY 2 org.eclipse.ui.workbench.texteditor 2 0 2011-12-17 17:04:50.062
> !MESSAGE Missing required bundle org.eclipse.ui_[3.5.0,4.0.0).
> !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.063
> !MESSAGE Bundle
> org.eclipse.ui.views.properties.tabbed_3.5.200.I20110201-0800 [64] was not
> resolved.
> !SUBENTRY 2 org.eclipse.ui.views.properties.tabbed 2 0 2011-12-17
> 17:04:50.063
> !MESSAGE Missing required bundle org.eclipse.ui.views_[3.2.0,4.0.0).
> !SUBENTRY 2 org.eclipse.ui.views.properties.tabbed 2 0 2011-12-17
> 17:04:50.063
> !MESSAGE Missing required bundle org.eclipse.ui_[3.3.0,4.0.0).
> !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.063
> !MESSAGE Bundle org.eclipse.platform_3.7.1.v201109091335 [67] was not
> resolved.
> !SUBENTRY 2 org.eclipse.platform 2 0 2011-12-17 17:04:50.063
> !MESSAGE Missing required bundle org.eclipse.ui.intro_[3.2.0,4.0.0).
> !SUBENTRY 2 org.eclipse.platform 2 0 2011-12-17 17:04:50.063
> !MESSAGE Missing optionally required bundle
> org.eclipse.ui.cheatsheets_[3.2.0,4.0.0).
> !SUBENTRY 2 org.eclipse.platform 2 0 2011-12-17 17:04:50.064
> !MESSAGE Missing optionally required bundle org.eclipse.ui_[3.2.0,4.0.0).
> !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.064
> !MESSAGE Bundle org.apache.lucene.analysis_2.9.1.v201101211721 [71] was
> not resolved.
> !SUBENTRY 2 org.apache.lucene.analysis 2 0 2011-12-17 17:04:50.064
> !MESSAGE Missing required bundle org.apache.lucene.core_[2.9.1,3.0.0).
> !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.064
> !MESSAGE Bundle org.eclipse.ltk.ui.refactoring_3.6.0.v20110505-0800 [78]
> was not resolved.
> !SUBENTRY 2 org.eclipse.ltk.ui.refactoring 2 0 2011-12-17 17:04:50.064
> !MESSAGE Missing required bundle org.eclipse.ui_[3.5.0,4.0.0).
> !SUBENTRY 2 org.eclipse.ltk.ui.refactoring 2 0 2011-12-17 17:04:50.064
> !MESSAGE Missing required bundle org.eclipse.ui.navigator_[3.3.200,4.0.0).
> !SUBENTRY 2 org.eclipse.ltk.ui.refactoring 2 0 2011-12-17 17:04:50.065
> !MESSAGE Missing required bundle org.eclipse.compare_[3.5.0,4.0.0).
> !SUBENTRY 2 org.eclipse.ltk.ui.refactoring 2 0 2011-12-17 17:04:50.065
> !MESSAGE Missing required bundle org.eclipse.team.ui_[3.4.100,4.0.0).
> 
> !ENTRY org.eclipse.osgi 4 0 2011-12-17 17:04:50.069
> !MESSAGE Application error
> !STACK 1
> java.lang.RuntimeException: No application id has been found.
>     at
> org.eclipse.equinox.internal.app.EclipseAppContainer.startDefaultApp(EclipseAppContainer.java:242)
>     at
> org.eclipse.equinox.internal.app.MainApplicationLauncher.run(MainApplicationLauncher.java:29)
>     at
> org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:110)
>     at
> org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:79)
>     at
> org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:344)
>     at
> org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:179)
>     at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
>     at
> sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
>     at
> sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
>     at java.lang.reflect.Method.invoke(Method.java:597)
>     at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:622)
>     at org.eclipse.equinox.launcher.Main.basicRun(Main.java:577)
>     at org.eclipse.equinox.launcher.Main.run(Main.java:1410)
>     at org.eclipse.equinox.launcher.Main.main(Main.java:1386)
> An error has occurred. See the log file
> 
> /Users/peterneubauer/Documents/workspaces/neoclipse/.metadata/.plugins/org.eclipse.pde.core/neoclipse.product/1324137888703.log.
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/neo4j/neoclipse/pull/5#issuecomment-3194674

## 

Thanks and Regards
N Radhakrishna Kalyan

peterneubauer commented:
No,
With the current code base.Anders?

Cheers

/peter

Sent from a device with crappy keyboard and autocorrect
On Dec 18, 2011 3:19 PM, "Radhakrishna Kalyan" <
reply@reply.github.com>
wrote:

> Are you trying after my commits?
> 
> On Sun, Dec 18, 2011 at 2:24 PM, Peter Neubauer <
> reply@reply.github.com
> 
> > wrote:
> > 
> > Guys,
> > I am trying to run the eclipse.product with Eclipse Indigo RCP, GEF-ZEST
> > installed, but get the stack trace below. any idea what is wrong?
> > 
> > [12/17/11 5:06:00 PM] Peter Neubauer: !MESSAGE Missing required bundle
> > org.eclipse.ui_[3.6.0,4.0.0).
> > !SUBENTRY 2 org.eclipse.ui.cheatsheets 2 0 2011-12-17 17:04:50.058
> > !MESSAGE Missing optionally required bundle
> > org.eclipse.help.base_[3.5.0,4.0.0).
> > !SUBENTRY 2 org.eclipse.ui.cheatsheets 2 0 2011-12-17 17:04:50.058
> > !MESSAGE Missing optionally required bundle
> > org.eclipse.help.ui_[3.5.0,4.0.0).
> > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.058
> > !MESSAGE Bundle org.eclipse.compare_3.5.201.R37x_v20110817-0800 [51] was
> > not resolved.
> > !SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.058
> > !MESSAGE Missing required bundle org.eclipse.ui_[3.5.0,4.0.0).
> > !SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.059
> > !MESSAGE Missing required bundle org.eclipse.ui.ide_[3.3.0,4.0.0).
> > !SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.059
> > !MESSAGE Missing required bundle org.eclipse.ui.views_[3.2.0,4.0.0).
> > !SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.059
> > !MESSAGE Missing required bundle
> > org.eclipse.ui.workbench.texteditor_[3.5.0,4.0.0).
> > !SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.059
> > !MESSAGE Missing required bundle org.eclipse.ui.editors_[3.5.0,4.0.0).
> > !SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.059
> > !MESSAGE Missing required bundle org.eclipse.compare.core_[3.5.0,4.0.0).
> > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.059
> > !MESSAGE Bundle org.eclipse.zest.core_1.3.0.v20110221-2050 [52] was not
> > resolved.
> > !SUBENTRY 2 org.eclipse.zest.core 2 0 2011-12-17 17:04:50.060
> > !MESSAGE Missing required bundle org.eclipse.ui_0.0.0.
> > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.060
> > !MESSAGE Bundle org.junit4_4.8.1.v20100525 [53] was not resolved.
> > !SUBENTRY 2 org.junit4 2 0 2011-12-17 17:04:50.060
> > !MESSAGE Missing required bundle org.hamcrest.core_1.1.0.
> > !SUBENTRY 2 org.junit4 2 0 2011-12-17 17:04:50.060
> > !MESSAGE Missing required bundle org.junit_4.8.1.
> > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.060
> > !MESSAGE Bundle org.eclipse.ui_3.7.0.I20110602-0100 [58] was not
> > resolved.
> > !SUBENTRY 2 org.eclipse.ui 2 0 2011-12-17 17:04:50.060
> > !MESSAGE Missing required bundle org.eclipse.ui.workbench_[3.5.0,4.0.0).
> > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.061
> > !MESSAGE Bundle org.apache.lucene_2.9.1.v201101211721 [60] was not
> > resolved.
> > !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.061
> > !MESSAGE Missing required bundle org.apache.lucene.core_[2.9.1,3.0.0).
> > !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.061
> > !MESSAGE Missing optionally required bundle
> > org.apache.lucene.analysis_[2.9.1,3.0.0).
> > !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.061
> > !MESSAGE Missing optionally required bundle
> > org.apache.lucene.highlighter_[2.9.1,3.0.0).
> > !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.061
> > !MESSAGE Missing optionally required bundle
> > org.apache.lucene.memory_[2.9.1,3.0.0).
> > !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.061
> > !MESSAGE Missing optionally required bundle
> > org.apache.lucene.queries_[2.9.1,3.0.0).
> > !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.062
> > !MESSAGE Missing optionally required bundle
> > org.apache.lucene.snowball_[2.9.1,3.0.0).
> > !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.062
> > !MESSAGE Missing optionally required bundle
> > org.apache.lucene.spellchecker_[2.9.1,3.0.0).
> > !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.062
> > !MESSAGE Missing optionally required bundle
> > org.apache.lucene.misc_[2.9.1,3.0.0).
> > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.062
> > !MESSAGE Bundle org.eclipse.ui.workbench.texteditor_3.7.0.v20110505-0800
> > [61] was not resolved.
> > !SUBENTRY 2 org.eclipse.ui.workbench.texteditor 2 0 2011-12-17
> > 17:04:50.062
> > !MESSAGE Missing required bundle org.eclipse.compare.core_[3.5.0,4.0.0).
> > !SUBENTRY 2 org.eclipse.ui.workbench.texteditor 2 0 2011-12-17
> > 17:04:50.062
> > !MESSAGE Missing required bundle org.eclipse.ui_[3.5.0,4.0.0).
> > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.063
> > !MESSAGE Bundle
> > org.eclipse.ui.views.properties.tabbed_3.5.200.I20110201-0800 [64] was
> > not
> > resolved.
> > !SUBENTRY 2 org.eclipse.ui.views.properties.tabbed 2 0 2011-12-17
> > 17:04:50.063
> > !MESSAGE Missing required bundle org.eclipse.ui.views_[3.2.0,4.0.0).
> > !SUBENTRY 2 org.eclipse.ui.views.properties.tabbed 2 0 2011-12-17
> > 17:04:50.063
> > !MESSAGE Missing required bundle org.eclipse.ui_[3.3.0,4.0.0).
> > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.063
> > !MESSAGE Bundle org.eclipse.platform_3.7.1.v201109091335 [67] was not
> > resolved.
> > !SUBENTRY 2 org.eclipse.platform 2 0 2011-12-17 17:04:50.063
> > !MESSAGE Missing required bundle org.eclipse.ui.intro_[3.2.0,4.0.0).
> > !SUBENTRY 2 org.eclipse.platform 2 0 2011-12-17 17:04:50.063
> > !MESSAGE Missing optionally required bundle
> > org.eclipse.ui.cheatsheets_[3.2.0,4.0.0).
> > !SUBENTRY 2 org.eclipse.platform 2 0 2011-12-17 17:04:50.064
> > !MESSAGE Missing optionally required bundle org.eclipse.ui_[3.2.0,4.0.0).
> > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.064
> > !MESSAGE Bundle org.apache.lucene.analysis_2.9.1.v201101211721 [71] was
> > not resolved.
> > !SUBENTRY 2 org.apache.lucene.analysis 2 0 2011-12-17 17:04:50.064
> > !MESSAGE Missing required bundle org.apache.lucene.core_[2.9.1,3.0.0).
> > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.064
> > !MESSAGE Bundle org.eclipse.ltk.ui.refactoring_3.6.0.v20110505-0800 [78]
> > was not resolved.
> > !SUBENTRY 2 org.eclipse.ltk.ui.refactoring 2 0 2011-12-17 17:04:50.064
> > !MESSAGE Missing required bundle org.eclipse.ui_[3.5.0,4.0.0).
> > !SUBENTRY 2 org.eclipse.ltk.ui.refactoring 2 0 2011-12-17 17:04:50.064
> > !MESSAGE Missing required bundle
> > org.eclipse.ui.navigator_[3.3.200,4.0.0).
> > !SUBENTRY 2 org.eclipse.ltk.ui.refactoring 2 0 2011-12-17 17:04:50.065
> > !MESSAGE Missing required bundle org.eclipse.compare_[3.5.0,4.0.0).
> > !SUBENTRY 2 org.eclipse.ltk.ui.refactoring 2 0 2011-12-17 17:04:50.065
> > !MESSAGE Missing required bundle org.eclipse.team.ui_[3.4.100,4.0.0).
> > 
> > !ENTRY org.eclipse.osgi 4 0 2011-12-17 17:04:50.069
> > !MESSAGE Application error
> > !STACK 1
> > java.lang.RuntimeException: No application id has been found.
> >     at
> > 
> > org.eclipse.equinox.internal.app.EclipseAppContainer.startDefaultApp(EclipseAppContainer.java:242)
> >     at
> > 
> > org.eclipse.equinox.internal.app.MainApplicationLauncher.run(MainApplicationLauncher.java:29)
> >     at
> > 
> > org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:110)
> >     at
> > 
> > org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:79)
> >     at
> > 
> > org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:344)
> >     at
> > 
> > org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:179)
> >     at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> >     at
> > 
> > sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
> >     at
> > 
> > sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
> >     at java.lang.reflect.Method.invoke(Method.java:597)
> >     at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:622)
> >     at org.eclipse.equinox.launcher.Main.basicRun(Main.java:577)
> >     at org.eclipse.equinox.launcher.Main.run(Main.java:1410)
> >     at org.eclipse.equinox.launcher.Main.main(Main.java:1386)
> > An error has occurred. See the log file
> > 
> > /Users/peterneubauer/Documents/workspaces/neoclipse/.metadata/.plugins/org.eclipse.pde.core/neoclipse.product/1324137888703.log.
> > 
> > ---
> > 
> > Reply to this email directly or view it on GitHub:
> > https://github.com/neo4j/neoclipse/pull/5#issuecomment-3194674
> 
> ## 
> 
> Thanks and Regards
> N Radhakrishna Kalyan
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/neo4j/neoclipse/pull/5#issuecomment-3194877

nrkkalyan commented:
I also got that error before, but I did the following steps:
- Run Configurations… -> Eclipse Application-> neoclipse.product

Under Main tab
used workspace location : ${workspace_loc:org.neo4j.neoclipse}

Under Plug-ins tab
Check marked 'Validate plugins automatically prior to launching'
and clicked 'Add Required Plig-ins' button

*And click run. Please try and let me know if you can able to run.

It worked when I did this for the first time, but second time I got some
other exception. I am trying to find the root cause.

Caused by: java.lang.ClassNotFoundException:
org.eclipse.ui.plugin.AbstractUIPlugin
    at
org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:513)
    at
org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:429)
    at
org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:417)
    at
org.eclipse.osgi.internal.baseadaptor.DefaultClassLoader.loadClass(DefaultClassLoader.java:107)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:247)
    ... 98 more

On Sun, Dec 18, 2011 at 3:55 PM, Peter Neubauer <
reply@reply.github.com

> wrote:
> 
> No,
> With the current code base.Anders?
> 
> Cheers
> 
> /peter
> 
> Sent from a device with crappy keyboard and autocorrect
> On Dec 18, 2011 3:19 PM, "Radhakrishna Kalyan" <
> reply@reply.github.com>
> wrote:
> 
> > Are you trying after my commits?
> > 
> > On Sun, Dec 18, 2011 at 2:24 PM, Peter Neubauer <
> > reply@reply.github.com
> > 
> > > wrote:
> > > 
> > > Guys,
> > > I am trying to run the eclipse.product with Eclipse Indigo RCP,
> > > GEF-ZEST
> > > installed, but get the stack trace below. any idea what is wrong?
> > > 
> > > [12/17/11 5:06:00 PM] Peter Neubauer: !MESSAGE Missing required bundle
> > > org.eclipse.ui_[3.6.0,4.0.0).
> > > !SUBENTRY 2 org.eclipse.ui.cheatsheets 2 0 2011-12-17 17:04:50.058
> > > !MESSAGE Missing optionally required bundle
> > > org.eclipse.help.base_[3.5.0,4.0.0).
> > > !SUBENTRY 2 org.eclipse.ui.cheatsheets 2 0 2011-12-17 17:04:50.058
> > > !MESSAGE Missing optionally required bundle
> > > org.eclipse.help.ui_[3.5.0,4.0.0).
> > > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.058
> > > !MESSAGE Bundle org.eclipse.compare_3.5.201.R37x_v20110817-0800 [51]
> > > was
> > > not resolved.
> > > !SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.058
> > > !MESSAGE Missing required bundle org.eclipse.ui_[3.5.0,4.0.0).
> > > !SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.059
> > > !MESSAGE Missing required bundle org.eclipse.ui.ide_[3.3.0,4.0.0).
> > > !SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.059
> > > !MESSAGE Missing required bundle org.eclipse.ui.views_[3.2.0,4.0.0).
> > > !SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.059
> > > !MESSAGE Missing required bundle
> > > org.eclipse.ui.workbench.texteditor_[3.5.0,4.0.0).
> > > !SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.059
> > > !MESSAGE Missing required bundle org.eclipse.ui.editors_[3.5.0,4.0.0).
> > > !SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.059
> > > !MESSAGE Missing required bundle
> > > org.eclipse.compare.core_[3.5.0,4.0.0).
> > > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.059
> > > !MESSAGE Bundle org.eclipse.zest.core_1.3.0.v20110221-2050 [52] was not
> > > resolved.
> > > !SUBENTRY 2 org.eclipse.zest.core 2 0 2011-12-17 17:04:50.060
> > > !MESSAGE Missing required bundle org.eclipse.ui_0.0.0.
> > > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.060
> > > !MESSAGE Bundle org.junit4_4.8.1.v20100525 [53] was not resolved.
> > > !SUBENTRY 2 org.junit4 2 0 2011-12-17 17:04:50.060
> > > !MESSAGE Missing required bundle org.hamcrest.core_1.1.0.
> > > !SUBENTRY 2 org.junit4 2 0 2011-12-17 17:04:50.060
> > > !MESSAGE Missing required bundle org.junit_4.8.1.
> > > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.060
> > > !MESSAGE Bundle org.eclipse.ui_3.7.0.I20110602-0100 [58] was not
> > > resolved.
> > > !SUBENTRY 2 org.eclipse.ui 2 0 2011-12-17 17:04:50.060
> > > !MESSAGE Missing required bundle
> > > org.eclipse.ui.workbench_[3.5.0,4.0.0).
> > > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.061
> > > !MESSAGE Bundle org.apache.lucene_2.9.1.v201101211721 [60] was not
> > > resolved.
> > > !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.061
> > > !MESSAGE Missing required bundle org.apache.lucene.core_[2.9.1,3.0.0).
> > > !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.061
> > > !MESSAGE Missing optionally required bundle
> > > org.apache.lucene.analysis_[2.9.1,3.0.0).
> > > !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.061
> > > !MESSAGE Missing optionally required bundle
> > > org.apache.lucene.highlighter_[2.9.1,3.0.0).
> > > !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.061
> > > !MESSAGE Missing optionally required bundle
> > > org.apache.lucene.memory_[2.9.1,3.0.0).
> > > !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.061
> > > !MESSAGE Missing optionally required bundle
> > > org.apache.lucene.queries_[2.9.1,3.0.0).
> > > !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.062
> > > !MESSAGE Missing optionally required bundle
> > > org.apache.lucene.snowball_[2.9.1,3.0.0).
> > > !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.062
> > > !MESSAGE Missing optionally required bundle
> > > org.apache.lucene.spellchecker_[2.9.1,3.0.0).
> > > !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.062
> > > !MESSAGE Missing optionally required bundle
> > > org.apache.lucene.misc_[2.9.1,3.0.0).
> > > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.062
> > > !MESSAGE Bundle
> > > org.eclipse.ui.workbench.texteditor_3.7.0.v20110505-0800
> > > [61] was not resolved.
> > > !SUBENTRY 2 org.eclipse.ui.workbench.texteditor 2 0 2011-12-17
> > > 17:04:50.062
> > > !MESSAGE Missing required bundle
> > > org.eclipse.compare.core_[3.5.0,4.0.0).
> > > !SUBENTRY 2 org.eclipse.ui.workbench.texteditor 2 0 2011-12-17
> > > 17:04:50.062
> > > !MESSAGE Missing required bundle org.eclipse.ui_[3.5.0,4.0.0).
> > > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.063
> > > !MESSAGE Bundle
> > > org.eclipse.ui.views.properties.tabbed_3.5.200.I20110201-0800 [64] was
> > > not
> > > resolved.
> > > !SUBENTRY 2 org.eclipse.ui.views.properties.tabbed 2 0 2011-12-17
> > > 17:04:50.063
> > > !MESSAGE Missing required bundle org.eclipse.ui.views_[3.2.0,4.0.0).
> > > !SUBENTRY 2 org.eclipse.ui.views.properties.tabbed 2 0 2011-12-17
> > > 17:04:50.063
> > > !MESSAGE Missing required bundle org.eclipse.ui_[3.3.0,4.0.0).
> > > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.063
> > > !MESSAGE Bundle org.eclipse.platform_3.7.1.v201109091335 [67] was not
> > > resolved.
> > > !SUBENTRY 2 org.eclipse.platform 2 0 2011-12-17 17:04:50.063
> > > !MESSAGE Missing required bundle org.eclipse.ui.intro_[3.2.0,4.0.0).
> > > !SUBENTRY 2 org.eclipse.platform 2 0 2011-12-17 17:04:50.063
> > > !MESSAGE Missing optionally required bundle
> > > org.eclipse.ui.cheatsheets_[3.2.0,4.0.0).
> > > !SUBENTRY 2 org.eclipse.platform 2 0 2011-12-17 17:04:50.064
> > > !MESSAGE Missing optionally required bundle
> > > org.eclipse.ui_[3.2.0,4.0.0).
> > > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.064
> > > !MESSAGE Bundle org.apache.lucene.analysis_2.9.1.v201101211721 [71] was
> > > not resolved.
> > > !SUBENTRY 2 org.apache.lucene.analysis 2 0 2011-12-17 17:04:50.064
> > > !MESSAGE Missing required bundle org.apache.lucene.core_[2.9.1,3.0.0).
> > > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.064
> > > !MESSAGE Bundle org.eclipse.ltk.ui.refactoring_3.6.0.v20110505-0800
> > > [78]
> > > was not resolved.
> > > !SUBENTRY 2 org.eclipse.ltk.ui.refactoring 2 0 2011-12-17 17:04:50.064
> > > !MESSAGE Missing required bundle org.eclipse.ui_[3.5.0,4.0.0).
> > > !SUBENTRY 2 org.eclipse.ltk.ui.refactoring 2 0 2011-12-17 17:04:50.064
> > > !MESSAGE Missing required bundle
> > > org.eclipse.ui.navigator_[3.3.200,4.0.0).
> > > !SUBENTRY 2 org.eclipse.ltk.ui.refactoring 2 0 2011-12-17 17:04:50.065
> > > !MESSAGE Missing required bundle org.eclipse.compare_[3.5.0,4.0.0).
> > > !SUBENTRY 2 org.eclipse.ltk.ui.refactoring 2 0 2011-12-17 17:04:50.065
> > > !MESSAGE Missing required bundle org.eclipse.team.ui_[3.4.100,4.0.0).
> > > 
> > > !ENTRY org.eclipse.osgi 4 0 2011-12-17 17:04:50.069
> > > !MESSAGE Application error
> > > !STACK 1
> > > java.lang.RuntimeException: No application id has been found.
> > >     at
> > 
> > org.eclipse.equinox.internal.app.EclipseAppContainer.startDefaultApp(EclipseAppContainer.java:242)
> > 
> > > ```
> > > at
> > > ```
> > 
> > org.eclipse.equinox.internal.app.MainApplicationLauncher.run(MainApplicationLauncher.java:29)
> > 
> > > ```
> > > at
> > > ```
> > 
> > org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:110)
> > 
> > > ```
> > > at
> > > ```
> > 
> > org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:79)
> > 
> > > ```
> > > at
> > > ```
> > 
> > org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:344)
> > 
> > > ```
> > > at
> > > ```
> > 
> > org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:179)
> > 
> > > ```
> > > at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> > > at
> > > ```
> > 
> > sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
> > 
> > > ```
> > > at
> > > ```
> > 
> > sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
> > 
> > > ```
> > > at java.lang.reflect.Method.invoke(Method.java:597)
> > > at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:622)
> > > at org.eclipse.equinox.launcher.Main.basicRun(Main.java:577)
> > > at org.eclipse.equinox.launcher.Main.run(Main.java:1410)
> > > at org.eclipse.equinox.launcher.Main.main(Main.java:1386)
> > > ```
> > > 
> > > An error has occurred. See the log file
> > 
> > /Users/peterneubauer/Documents/workspaces/neoclipse/.metadata/.plugins/org.eclipse.pde.core/neoclipse.product/1324137888703.log.
> > 
> > > ---
> > > 
> > > Reply to this email directly or view it on GitHub:
> > > https://github.com/neo4j/neoclipse/pull/5#issuecomment-3194674
> > 
> > ## 
> > 
> > Thanks and Regards
> > N Radhakrishna Kalyan
> > 
> > ---
> > 
> > Reply to this email directly or view it on GitHub:
> > https://github.com/neo4j/neoclipse/pull/5#issuecomment-3194877
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/neo4j/neoclipse/pull/5#issuecomment-3195058

## 

Thanks and Regards
N Radhakrishna Kalyan

peterneubauer commented:
Ha! Works.
now trying out your changes. thanks!

/peter

On Sun, Dec 18, 2011 at 4:53 PM, Radhakrishna Kalyan
reply@reply.github.com
wrote:

> I also got that error before, but I did the following steps:
> - Run Configurations… -> Eclipse Application-> neoclipse.product
> 
> Under Main tab
> used workspace location : ${workspace_loc:org.neo4j.neoclipse}
> 
> Under Plug-ins tab
> Check marked 'Validate plugins automatically prior to launching'
> and clicked 'Add Required Plig-ins' button
> 
> *And click run. Please try and let me know if you can able to run.
> 
> It worked when I did this for the first time, but second time I got some
> other exception. I am trying to find the root cause.
> 
> Caused by: java.lang.ClassNotFoundException:
> org.eclipse.ui.plugin.AbstractUIPlugin
>    at
> org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:513)
>    at
> org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:429)
>    at
> org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:417)
>    at
> org.eclipse.osgi.internal.baseadaptor.DefaultClassLoader.loadClass(DefaultClassLoader.java:107)
>    at java.lang.ClassLoader.loadClass(ClassLoader.java:247)
>    ... 98 more
> 
> On Sun, Dec 18, 2011 at 3:55 PM, Peter Neubauer <
> reply@reply.github.com
> 
> > wrote:
> > 
> > No,
> > With the current code base.Anders?
> > 
> > Cheers
> > 
> > /peter
> > 
> > Sent from a device with crappy keyboard and autocorrect
> > On Dec 18, 2011 3:19 PM, "Radhakrishna Kalyan" <
> > reply@reply.github.com>
> > wrote:
> > 
> > > Are you trying after my commits?
> > > 
> > > On Sun, Dec 18, 2011 at 2:24 PM, Peter Neubauer <
> > > reply@reply.github.com
> > > 
> > > > wrote:
> > > > 
> > > > Guys,
> > > > I am trying to run the eclipse.product with Eclipse Indigo RCP,
> > > > GEF-ZEST
> > > > installed, but get the stack trace below. any idea what is wrong?
> > > > 
> > > > [12/17/11 5:06:00 PM] Peter Neubauer: !MESSAGE Missing required bundle
> > > > org.eclipse.ui_[3.6.0,4.0.0).
> > > > !SUBENTRY 2 org.eclipse.ui.cheatsheets 2 0 2011-12-17 17:04:50.058
> > > > !MESSAGE Missing optionally required bundle
> > > > org.eclipse.help.base_[3.5.0,4.0.0).
> > > > !SUBENTRY 2 org.eclipse.ui.cheatsheets 2 0 2011-12-17 17:04:50.058
> > > > !MESSAGE Missing optionally required bundle
> > > > org.eclipse.help.ui_[3.5.0,4.0.0).
> > > > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.058
> > > > !MESSAGE Bundle org.eclipse.compare_3.5.201.R37x_v20110817-0800 [51]
> > > > was
> > > > not resolved.
> > > > !SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.058
> > > > !MESSAGE Missing required bundle org.eclipse.ui_[3.5.0,4.0.0).
> > > > !SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.059
> > > > !MESSAGE Missing required bundle org.eclipse.ui.ide_[3.3.0,4.0.0).
> > > > !SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.059
> > > > !MESSAGE Missing required bundle org.eclipse.ui.views_[3.2.0,4.0.0).
> > > > !SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.059
> > > > !MESSAGE Missing required bundle
> > > > org.eclipse.ui.workbench.texteditor_[3.5.0,4.0.0).
> > > > !SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.059
> > > > !MESSAGE Missing required bundle org.eclipse.ui.editors_[3.5.0,4.0.0).
> > > > !SUBENTRY 2 org.eclipse.compare 2 0 2011-12-17 17:04:50.059
> > > > !MESSAGE Missing required bundle
> > > > org.eclipse.compare.core_[3.5.0,4.0.0).
> > > > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.059
> > > > !MESSAGE Bundle org.eclipse.zest.core_1.3.0.v20110221-2050 [52] was not
> > > > resolved.
> > > > !SUBENTRY 2 org.eclipse.zest.core 2 0 2011-12-17 17:04:50.060
> > > > !MESSAGE Missing required bundle org.eclipse.ui_0.0.0.
> > > > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.060
> > > > !MESSAGE Bundle org.junit4_4.8.1.v20100525 [53] was not resolved.
> > > > !SUBENTRY 2 org.junit4 2 0 2011-12-17 17:04:50.060
> > > > !MESSAGE Missing required bundle org.hamcrest.core_1.1.0.
> > > > !SUBENTRY 2 org.junit4 2 0 2011-12-17 17:04:50.060
> > > > !MESSAGE Missing required bundle org.junit_4.8.1.
> > > > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.060
> > > > !MESSAGE Bundle org.eclipse.ui_3.7.0.I20110602-0100 [58] was not
> > > > resolved.
> > > > !SUBENTRY 2 org.eclipse.ui 2 0 2011-12-17 17:04:50.060
> > > > !MESSAGE Missing required bundle
> > > > org.eclipse.ui.workbench_[3.5.0,4.0.0).
> > > > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.061
> > > > !MESSAGE Bundle org.apache.lucene_2.9.1.v201101211721 [60] was not
> > > > resolved.
> > > > !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.061
> > > > !MESSAGE Missing required bundle org.apache.lucene.core_[2.9.1,3.0.0).
> > > > !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.061
> > > > !MESSAGE Missing optionally required bundle
> > > > org.apache.lucene.analysis_[2.9.1,3.0.0).
> > > > !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.061
> > > > !MESSAGE Missing optionally required bundle
> > > > org.apache.lucene.highlighter_[2.9.1,3.0.0).
> > > > !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.061
> > > > !MESSAGE Missing optionally required bundle
> > > > org.apache.lucene.memory_[2.9.1,3.0.0).
> > > > !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.061
> > > > !MESSAGE Missing optionally required bundle
> > > > org.apache.lucene.queries_[2.9.1,3.0.0).
> > > > !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.062
> > > > !MESSAGE Missing optionally required bundle
> > > > org.apache.lucene.snowball_[2.9.1,3.0.0).
> > > > !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.062
> > > > !MESSAGE Missing optionally required bundle
> > > > org.apache.lucene.spellchecker_[2.9.1,3.0.0).
> > > > !SUBENTRY 2 org.apache.lucene 2 0 2011-12-17 17:04:50.062
> > > > !MESSAGE Missing optionally required bundle
> > > > org.apache.lucene.misc_[2.9.1,3.0.0).
> > > > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.062
> > > > !MESSAGE Bundle
> > > > org.eclipse.ui.workbench.texteditor_3.7.0.v20110505-0800
> > > > [61] was not resolved.
> > > > !SUBENTRY 2 org.eclipse.ui.workbench.texteditor 2 0 2011-12-17
> > > > 17:04:50.062
> > > > !MESSAGE Missing required bundle
> > > > org.eclipse.compare.core_[3.5.0,4.0.0).
> > > > !SUBENTRY 2 org.eclipse.ui.workbench.texteditor 2 0 2011-12-17
> > > > 17:04:50.062
> > > > !MESSAGE Missing required bundle org.eclipse.ui_[3.5.0,4.0.0).
> > > > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.063
> > > > !MESSAGE Bundle
> > > > org.eclipse.ui.views.properties.tabbed_3.5.200.I20110201-0800 [64] was
> > > > not
> > > > resolved.
> > > > !SUBENTRY 2 org.eclipse.ui.views.properties.tabbed 2 0 2011-12-17
> > > > 17:04:50.063
> > > > !MESSAGE Missing required bundle org.eclipse.ui.views_[3.2.0,4.0.0).
> > > > !SUBENTRY 2 org.eclipse.ui.views.properties.tabbed 2 0 2011-12-17
> > > > 17:04:50.063
> > > > !MESSAGE Missing required bundle org.eclipse.ui_[3.3.0,4.0.0).
> > > > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.063
> > > > !MESSAGE Bundle org.eclipse.platform_3.7.1.v201109091335 [67] was not
> > > > resolved.
> > > > !SUBENTRY 2 org.eclipse.platform 2 0 2011-12-17 17:04:50.063
> > > > !MESSAGE Missing required bundle org.eclipse.ui.intro_[3.2.0,4.0.0).
> > > > !SUBENTRY 2 org.eclipse.platform 2 0 2011-12-17 17:04:50.063
> > > > !MESSAGE Missing optionally required bundle
> > > > org.eclipse.ui.cheatsheets_[3.2.0,4.0.0).
> > > > !SUBENTRY 2 org.eclipse.platform 2 0 2011-12-17 17:04:50.064
> > > > !MESSAGE Missing optionally required bundle
> > > > org.eclipse.ui_[3.2.0,4.0.0).
> > > > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.064
> > > > !MESSAGE Bundle org.apache.lucene.analysis_2.9.1.v201101211721 [71] was
> > > > not resolved.
> > > > !SUBENTRY 2 org.apache.lucene.analysis 2 0 2011-12-17 17:04:50.064
> > > > !MESSAGE Missing required bundle org.apache.lucene.core_[2.9.1,3.0.0).
> > > > !SUBENTRY 1 org.eclipse.osgi 2 0 2011-12-17 17:04:50.064
> > > > !MESSAGE Bundle org.eclipse.ltk.ui.refactoring_3.6.0.v20110505-0800
> > > > [78]
> > > > was not resolved.
> > > > !SUBENTRY 2 org.eclipse.ltk.ui.refactoring 2 0 2011-12-17 17:04:50.064
> > > > !MESSAGE Missing required bundle org.eclipse.ui_[3.5.0,4.0.0).
> > > > !SUBENTRY 2 org.eclipse.ltk.ui.refactoring 2 0 2011-12-17 17:04:50.064
> > > > !MESSAGE Missing required bundle
> > > > org.eclipse.ui.navigator_[3.3.200,4.0.0).
> > > > !SUBENTRY 2 org.eclipse.ltk.ui.refactoring 2 0 2011-12-17 17:04:50.065
> > > > !MESSAGE Missing required bundle org.eclipse.compare_[3.5.0,4.0.0).
> > > > !SUBENTRY 2 org.eclipse.ltk.ui.refactoring 2 0 2011-12-17 17:04:50.065
> > > > !MESSAGE Missing required bundle org.eclipse.team.ui_[3.4.100,4.0.0).
> > > > 
> > > > !ENTRY org.eclipse.osgi 4 0 2011-12-17 17:04:50.069
> > > > !MESSAGE Application error
> > > > !STACK 1
> > > > java.lang.RuntimeException: No application id has been found.
> > > >     at
> > > 
> > > org.eclipse.equinox.internal.app.EclipseAppContainer.startDefaultApp(EclipseAppContainer.java:242)
> > > 
> > > >     at
> > > 
> > > org.eclipse.equinox.internal.app.MainApplicationLauncher.run(MainApplicationLauncher.java:29)
> > > 
> > > >     at
> > > 
> > > org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:110)
> > > 
> > > >     at
> > > 
> > > org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:79)
> > > 
> > > >     at
> > > 
> > > org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:344)
> > > 
> > > >     at
> > > 
> > > org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:179)
> > > 
> > > >     at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> > > >     at
> > > 
> > > sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
> > > 
> > > >     at
> > > 
> > > sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
> > > 
> > > >     at java.lang.reflect.Method.invoke(Method.java:597)
> > > >     at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:622)
> > > >     at org.eclipse.equinox.launcher.Main.basicRun(Main.java:577)
> > > >     at org.eclipse.equinox.launcher.Main.run(Main.java:1410)
> > > >     at org.eclipse.equinox.launcher.Main.main(Main.java:1386)
> > > > An error has occurred. See the log file
> > > 
> > > /Users/peterneubauer/Documents/workspaces/neoclipse/.metadata/.plugins/org.eclipse.pde.core/neoclipse.product/1324137888703.log.
> > > 
> > > > ---
> > > > 
> > > > Reply to this email directly or view it on GitHub:
> > > > https://github.com/neo4j/neoclipse/pull/5#issuecomment-3194674
> > > 
> > > ## 
> > > 
> > > Thanks and Regards
> > > N Radhakrishna Kalyan
> > > 
> > > ---
> > > 
> > > Reply to this email directly or view it on GitHub:
> > > https://github.com/neo4j/neoclipse/pull/5#issuecomment-3194877
> > 
> > ---
> > 
> > Reply to this email directly or view it on GitHub:
> > https://github.com/neo4j/neoclipse/pull/5#issuecomment-3195058
> 
> ## 
> 
> Thanks and Regards
> N Radhakrishna Kalyan
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/neo4j/neoclipse/pull/5#issuecomment-3195341

peterneubauer commented:
This looked nice, thanks a lot for the pull request! Do you have any other things you would like to see in Neoclipse?

nrkkalyan commented:
Hi,

ya, i am working on that, as an issue raised by you.
Trying to add a connections view like
http://www.sqlexplorer.org/connections.php

More over I have also plans to automate most of the code generations using
the plugin.
Hope you like.

Kalyan

On Sun, Dec 18, 2011 at 5:10 PM, Peter Neubauer <
reply@reply.github.com

> wrote:
> 
> This looked nice, thanks a lot for the pull request! Do you have any other
> things you would like to see in Neoclipse?
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/neo4j/neoclipse/pull/5#issuecomment-3195414

## 

Thanks and Regards
N Radhakrishna Kalyan

nawroth commented:
I wanted a connections view for ages now, would be great!
